### PR TITLE
refactor: dashboard homepage and some visible changes

### LINKED ISSUE #1115: Download existing / running experiments YML
## Feature Request

**Is your feature request related to a problem? Please describe:**
I have difficulty explaining to my colleagues how Chaos Experiments are structured as YML in Chaos Mesh. Since you can import experiments directly as YML when creating them on the dashboard, we would like to create our own examples. These should be able to be derived from running and functioning experiments according to our requirements. At the moment this is not yet possible.

**Describe the feature you'd like:**
I would like to be able to download/export the YML of the experiment in the detailed view of a running experiment in the dashboard.
I want to be able to manage this in my own SCM and share it with other teams as a basic script so that the teams have it easier to integrate chaos engineering into their daily work and to use the dashboard.

**Describe alternatives you've considered:**
none

**Teachability, Documentation, Adoption, Migration Strategy:**
On the "Update" button in the dashboard, you could add an export / download button, with which you can download the YML of an experiment. Similarly, this is e.g. possible in the Kubernetes dashboard in the detailed view of deployments. There you can view, edit and also download the YML live.

### PR DESCRIPTION
**Pull request by g1eny0ung**
### What problem does this PR solve?

This PR introduces several changes (from #1283) that will be effected in `v1.1.0`:

- [x] (refactor) Make the homepage more like a dashboard
- [x] (feat) Tourist Guide on home page
- [x] (feat) Predefined experiments (Can be store in the IndexedDB)
- [x] (chore) Security Mode support in UI
- [x] (feat) Download experiment YAML definition (Resolve #1115)
- [x] (feat) Embed Swagger API
- [x] (chore) Delete archive experiments by UI interface #1272

And also, this PR has some minor bug fixes:

- Can't view `DNSChaos` archives
- Repeatedly request `chaos-available-namespaces`
- Some bug fixes
- Styles optimized

Some extra fixes were open below. When it's merged in master. Some functions will work as normal.

- #1261
- #1323

### What is changed and how does it work?

In addition to the above changes, the dashboard UI has a layout change to adapt the recent features.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [x] Need to update the documentation

I will follow up this PR https://github.com/chaos-mesh/website/pull/24 to make sure its content is up to date.

### Does this PR introduce a user-facing change?

Xnip2020-12-17_17-15-30

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
Very beautiful

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

File: ui/README.md [hunk 1]
```diff
@@ -1,45 +1,151 @@
...
+- **From local server**
+
+  There have two ways to run chaos-dashboard server in your terminal:
+
+  - `go run cmd/chaos-dashboard/main.go`
+  - `SWAGGER=1 make chaos-dashboard && ./bin/chaos-dashboard`
+
+  One is real-time, the other needs to be compiled before use. The compiled bundle the extra Swagger API HTML into the binary file.
+
+  For the other usage of `make xxx` that please refer the [Makefile](https://github.com/chaos-mesh/chaos-mesh/blob/master/Makefile).
```
Comment (fewdan) [inline ui/README.md:87]: It seems to be changed to "please refer to the ...".

Comment (g1eny0ung) [reply ui/README.md]: Updated.~

**Review by fewdan — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]

**Pull request merged**